### DISCUSSION TITLE
I broke my database – now what?

### DISCUSSION THREAD

**Discussion by mstade** [supabase/supabase#1288] [Questions] [answered]
Hey all!

So I followed someones advice and executed `vacuum full analyze` on my database, and now it's broken. The query ran and said there was an error with the database, and since then all the things in the supabase UI that has anything to do with the database – table editor, database settings etc. – just says loading and then after a while it just says error, or displays this magnificent picture:

![](https://app.supabase.io/img/eastwood-fatal-error.svg)

Clearly, I messed up.

This is just a dev thing, so there's absolutely zero panic, but does this mean that project is now borked and I should try and delete it and make a new one? I'm mostly asking because it'll take a me a bit of time to recreate what's lost. It can all be recreated, it's just gonna take some time.

For anyone looking to understand why I ran that query – I've been doing some work on an import script that inserts quite a few rows, only to then delete it all when testing a new run. I suppose you could say I had a bit of a YOLO moment. Lesson learned... 🤪

**Comment by inian** [comment, marked as answer]
Hi Marcus, I was just looking at your server cos of an alert. A database restart fixed it (We are working on a way for you to do this from the dashboard). 

Not sure what the issue was, probably that command locked up the database fully?

**Reply by mstade** [reply]
Amazing support – thank you! 🙏 

I don't recall the specifics of the error, but I'm glad it was an easy fix. Saved me some time for sure! Thank you so much!
